modules/ROOT/pages/tokenization-example.adoc

```diff
@@ -242,19 +252,24 @@ The following is an example of bad detokenization:
 curl ­v ­-k ­­--resolve token2mule.ic.intel.com:443:10.230.36.230 https://token2mule.ic.intel.com/tb/v1/detokenization ­-X POST ­-H "Content­type: application/json" ­­data '[{"data": "597­74­8102­­­­­­­­sdsdsdsdsdsdsdsds", "format": "ssndemo"}]' ­-k ­­--user test:test
 ----
 
+You should receive a response similar to the below:
+
 ----
 HTTP/1.1 422 Unprocessable Entity
-[{"data":"","status":"failure","errorcode":1380,"error":
-"The social security number is invalid. It contains [26] characters.
+[{"data":"","status":"failure","errorcode":1380,"error":"The social security number is invalid.
+It contains [26] characters.
 A social security number must have the format ###-##-#### where # represents a decimal digit."}]
 ----
 
+
 The following is an example of bad tokenization JSON data stopped by Api Gateway protection:
 
 ----
 curl ­v ­-k ­­--resolve token2mule.ic.intel.com:443:10.230.36.230 https://token2mule.ic.intel.com/tb/v1/detokenization ­-X POST -­H "Content­type: application/json" ­­data '[{{{}{{{}]]"data": "597­74­8102­­­­­­­­sdsdsdsdsdsdsdsds", "format": "ssndemo"}]' ­-k ­­--user test:test
 ----
 
+You should receive a response similar to the below:
```

Comment:
"similar to the following" or "similar to this example"? (global, I'm not sure I caught all of these)